moveit_ros/moveit_servo/test/pose_tracking_test.cpp

```diff
@@ -135,15 +180,14 @@ TEST_F(PoseTrackingFixture, OutgoingMsgTest)
     size_t msg_count = 0;
     while (++msg_count < 100)
     {
-      target_pose_pub_.publish(target_pose);
-      ros::Duration(0.01).sleep();
+      target_pose_pub_->publish(target_pose);
+      std::this_thread::sleep_for(std::chrono::milliseconds(10));;
     }
   });
 
-  ros::Duration(ROS_PUB_SUB_DELAY).sleep();
+  std::this_thread::sleep_for(std::chrono::milliseconds(ROS_PUB_SUB_DELAY*100));
```

Comment:
```suggestion
  std::this_thread::sleep_for(std::chrono::seconds(ROS_PUB_SUB_DELAY));
```

Instead of doing the conversion manually with a multiply, you can just use the seconds type.